### ISSUE TITLE
Menyoo Clothing problem?

### ISSUE THREAD

**Issue by AmazyCrazy** [MAFINS/MenyooSP#586]
Hi, my game is completely up to date and so far working fine accept that in Menyoo when scrolling forward through the clothes it only stops at the latest dlc clothing and you can't choose any addon clothing that you have installed unless you go backwards then it works?

I've swear that I've seen this in the past but can't remember what the fix was?

Thanks in advance :)

**Comment by mistahwolf** [comment]
Does your game crash when that happens? It does on my system.

**Comment by AmazyCrazy** [comment]
@mistahwolf Yes it did the simple solution is to just copy the mp2023_01 rpf to your mods folder then go in and delete all of the ydd's the have a white placeholder when you view them. Please note that if you use the mp clothing add-on by HeySickThatsMe you will have to scroll backwards in Menyoo to view all of your added content.

Hope this helps :)

**Comment by The-UnXplained** [comment]
http://www.gtamods4you.com/

**Comment by ColbyDash** [comment]
> @mistahwolf Yes it did the simple solution is to just copy the mp2023_01 rpf to your mods folder then go in and delete all of the ydd's the have a white placeholder when you view them. Please note that if you use the mp clothing add-on by HeySickThatsMe you will have to scroll backwards in Menyoo to view all of your added content.
> 
> Hope this helps :)

This doesn't work for me unfortunately regardless of which trainer I use and I am not even using the mp clothing add-on either. :p
